NumLib/Extrapolation/Extrapolator.h

```diff
@@ -33,6 +33,15 @@ class Extrapolator
         std::vector<GlobalVector*> const& x,
         std::vector<NumLib::LocalToGlobalIndexMap const*> const& dof_table) = 0;
 
+    //! Extrapolates post-reaction process solutions from the given local
+    //! assemblers.
```

Comment:
It's better if NumLib does not include any process specific nomenclature, like post-reaction